src/components/L2/Headers/AdvancedOptions.tsx

```diff
@@ -11,6 +12,15 @@ interface AdvancedOptionsProps {
   }[];
 }
 
+// Hide this input completely
+const HiddenInput = styled.input`
```

Comment:
There's a `HiddenInput` exported from `src/forms/inputs`, and typically I would encourage reuse. But since we're eventually moving away from Indigo, fine by me.